db/schema.rb

```diff
@@ -44,6 +54,7 @@
     t.string   "last_sign_in_ip"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "name"
   end
 
   add_index "users", ["email"], name: "index_users_on_email", unique: true
```

Comment:
Yeah, this was straight copied from something in Khan Burmese. To be honest, I'm not actually sure how indexing works, but I'm just assuming it's beneficial.